src/main/webapp/availability.js

```diff
@@ -38,3 +38,17 @@ function browserTimezoneOffset() {
 function availabilityTableDiv() {
   return document.getElementById('table-container');
 }
+
+function updateAvailability() {
+  let selectedSlots = document.getElementsByClassName('table-success');
```

Comment:
Getting the elements by a class used for formatting is fragile. If anyone ever added a new table that used this class to this page, this code would break. You should probably add an additional class of your own devising (like `selected-time-slot`) to each selected element in addition to `table-success`.

Reply:
Done